src/test/func/two_alloc_types/main.cc

```diff
@@ -42,6 +47,10 @@ int main()
   oe_end = (uint8_t*)oe_base + size;
   std::cout << "Allocated region " << oe_base << " - " << oe_end << std::endl;
 
+  // Call these functions to trigger asserts if the cast-to-self doesn't work.
+  enclave_snmalloc_pagemap_global_get(nullptr);

```

Comment:
Doesn't passing nullptr, mean this will not do a check?

Reply:
Ooops, yes.